src/poliastro/atmosphere/isa.py

```diff
@@ -0,0 +1,176 @@
+""" International Standard Atmosphere model defined in ICAO Doc 7488/3. """
+
+import numpy as np
+from astropy import units as u
+from astropy.constants import R as R_u
+
+from poliastro.atmosphere.util import (
+    geometric_to_geopotential,
+    geopotential_to_geometric,
+    gravity,
+)
+
+# ATMOSPHERIC GASES PROPERTIES
+# TODO: # Use IAU values? Add in poliastro.constants?
+M0_air = 28.964420 * u.kg / u.kmol
+R_air = R_u / M0_air
+
+# GEOPOTENTIAL LAYERS PROPERTIES (TABLE D)
+Hb_layers = [0.00, 11.00, 20.00, 32.00, 47.00, 51.00, 71.00, 80.00] * u.km
+Tb_layers = [288.15, 216.65, 216.65, 228.65, 270.65, 270.65, 214.65, 196.65] * u.K
+pb_layers = [
+    1.01325e3,
+    2.26320e2,
+    5.47487e1,
+    8.68014e0,
+    1.10906e0,
+    6.69384e-1,
+    3.95639e-2,
+    8.86272e-3,
+] * u.hPa
+beta_set = [-6.50, 0.00, 1.00, 2.80, 0.00, -2.80, -2.00, 0.00] * u.K / u.km
+
+
+def _check_alt_limit(h):
+    """ Checks if a valid value of geometric altitude has been passed. """
+
+    if h < 0 * u.km or h > 80 * u.km:
+        raise ValueError(
+            "ISA model is only valid for geometric altitudes in range [0, 80] km."
+        )
+
+
+def _get_base_parameters(alt, geopot=False, gradient=True):
+    """ Returns appropiate temperature gradient.
+
+    Parameters
+    ----------
+    H: ~astropy.units.Quantity
+        Geopotential altitude.
+
+    Returns
+    -------
+    beta: ~astropy.units.Quantity
+        Temperature gradient.
+    """
+
+    # If required, convert to geopotential height
+    if not geopot:
+        _check_alt_limit(alt)
+        H = geometric_to_geopotential(alt)
+    else:
+        H = alt
+
+    for i, Hb in enumerate(Hb_layers):
+        # Return appropiate base height and temeperature
+        if Hb > H:
+            Hb = Hb_layers[i - 1]
+            Tb = Tb_layers[i - 1]
+            pb = pb_layers[i - 1]
+            if gradient:
+                beta = beta_set[i - 1]
+            else:
+                beta = 0 * u.K / u.km
+
+            return H, Hb, Tb, pb, beta
+
+        elif H == Hb:
+            # We are at layer. Return those values
+            Tb = Tb_layers[i]
+            pb = pb_layers[i]
+            if gradient:
+                beta = beta_set[i]
+            else:
+                beta = 0 * u.K / u.km
+
+            return H, Hb, Tb, pb, beta
+        else:
```

Comment:
This `else: pass` is not needed, I think

Reply:
With the class implementation, this has been simplified. It's not necessary anymore :+1: